RBS.Auth.WebApi/RBS.Auth.WebApi/Controllers/AuthController.cs

```diff
@@ -1,46 +1,62 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Options;
 using RBS.Auth.Common;
+using RBS.Auth.Common.Models;
 using RBS.Auth.Services.Interfaces.Authenticate;
 using RBS.Auth.Services.Interfaces.Tokens;
 using RBS.Auth.WebApi.Models;
 
-namespace RBS.Auth.WebApi.Controllers
+namespace RBS.Auth.WebApi.Controllers;
+
+[Route("api/auth")]
+[ApiController]
+public class AuthController : ControllerBase
 {
-    [Route("api/[controller]")]
-    [ApiController]
-    public class AuthController : ControllerBase
+    private readonly IAuthenticateService _authenticateService;
+    private readonly IOptions<AuthOptions> _authOptions;
+    private readonly IJwtTokenService _jwtTokenService;
+    private readonly IMapper _mapper;
+
+    public AuthController(IOptions<AuthOptions> authOptions,
+        IJwtTokenService jwtTokenService,
+        IAuthenticateService authenticateService,
+        IMapper mapper)
     {
-        private readonly IOptions<AuthOptions> _authOptions;
-        private readonly IJWTTokenService _jwtTokenService;
-        private readonly IAuthenticateService _authenticateService;
+        _authOptions = authOptions;
+        _jwtTokenService = jwtTokenService;
+        _authenticateService = authenticateService;
+        _mapper = mapper;
+    }
 
-        public AuthController(IOptions<AuthOptions> authOptions,
-            IJWTTokenService jwtTokenService,
-            IAuthenticateService authenticateService)
-        {
-            _authOptions = authOptions;
-            _jwtTokenService = jwtTokenService;
-            _authenticateService = authenticateService;
-        }
+    [Route("login")]
+    [HttpPost]
+    public IActionResult Login(LoginRequest request)
```

Comment:
Consider to use Async methods in controllers and services

Reply:
Done